docs/rabbitmq_acl_configure.sh

```diff
@@ -0,0 +1,90 @@
+#
+# This file creates individual AMQP accounts for each Boulder component,
+# and sets restrictive access controls on those accounts.
+#
+# You can use this tool without any configuration to produce users named
+# [am, ca, sa, ra, va, wfe, ocsp-updater] which all have the password "guest".
+# You can also customize this tool by creating a config file that will be
+# sourced. By default this file is obtained from $HOME/.rabbitmq_config, but
+# you can override the config file path using the environment variable
+# RABBITMQ_ACL_CONFIG, such as:
+#
+# $ RABBITMQ_ACL_CONFIG=myconfig ./rabbitmq_acl_configure.sh
+
+# VARIABLES
+PORT=15672
+HOST=localhost
+VHOST="/"
+EXTRA=""
+RABBIT_ADMIN=$(which rabbitmqadmin)
+
+# USER NAMES
+USER_BOULDER_AM="am"
+USER_BOULDER_CA="ca"
+USER_BOULDER_SA="sa"
+USER_BOULDER_RA="ra"
+USER_BOULDER_VA="va"
+USER_BOULDER_WFE="wfe"
+USER_BOULDER_OCSP="ocsp-updater"
+
+# PASSWORDS
+PASS_BOULDER_AM="guest"
+PASS_BOULDER_CA="guest"
+PASS_BOULDER_SA="guest"
+PASS_BOULDER_RA="guest"
+PASS_BOULDER_VA="guest"
+PASS_BOULDER_WFE="guest"
+PASS_BOULDER_OCSP="guest"
+
+# To use different options, you should create an override
+# file with whatever changes you want for the above variables
+RABBITMQ_ACL_CONFIG=${RABBITMQ_ACL_CONFIG:-$HOME/.rabbitmq_config}
+
+if [ -r "${RABBITMQ_ACL_CONFIG}" ] ; then
+  echo "Loading overrides from ${RABBITMQ_ACL_CONFIG}..."
+  source "${RABBITMQ_ACL_CONFIG}"
+fi
+
+if ! [ -x "${RABBIT_ADMIN}" ] ; then
+  echo "Could not locate rabbitmqadmin; please set RABBIT_ADMIN in your ${RABBITMQ_ACL_CONFIG} file."
+  exit 1
+fi
+
+run() {
+  echo $*
+  $*
+}
+
+admin() {
+  run ${RABBIT_ADMIN} -H ${HOST} -P ${PORT} -V ${VHOST} ${EXTRA} $*
+}
+
+admin declare queue name="Monitor" durable=false
+admin declare queue name="CA.server" durable=false
+admin declare queue name="SA.server" durable=false
+admin declare queue name="RA.server" durable=false
+admin declare queue name="VA.server" durable=false
+
+admin declare exchange name="boulder" type=topic durable=false
+admin declare binding source="boulder" destination="Monitor" routing_key="#"
+
+admin declare user name=${USER_BOULDER_AM} password=${PASS_BOULDER_AM} tags=""
+admin declare user name=${USER_BOULDER_CA} password=${PASS_BOULDER_CA} tags=""
+admin declare user name=${USER_BOULDER_SA} password=${PASS_BOULDER_SA} tags=""
+admin declare user name=${USER_BOULDER_RA} password=${PASS_BOULDER_RA} tags=""
+admin declare user name=${USER_BOULDER_VA} password=${PASS_BOULDER_VA} tags=""
+admin declare user name=${USER_BOULDER_WFE} password=${PASS_BOULDER_WFE} tags=""
+admin declare user name=${USER_BOULDER_OCSP} password=${PASS_BOULDER_OCSP} tags=""
+
+admin declare permission vhost=${VHOST} user=${USER_BOULDER_AM} configure="^$" write="^$" read="^Monitor$"
+admin declare permission vhost=${VHOST} user=${USER_BOULDER_VA} configure="^(VA\.server|VA->.*)$" write="^(boulder|VA\.server|VA->.*)$" read="^(boulder|VA\.server|VA->.*)$"
```

Comment:
These permissions declare that each user (except AM) has read/write to resources matching 'boulder'. It seems like matching includes the exchange name as well as the queue name, and all of these queues are on the 'boulder' exchange, right? So it seems like each user has read/write on all queues, which seems incorrect. My guess is that I'm reading the configuration incorrectly, so please let me know what I'm missing.

Also, why do the various servers need configure permission on their own queues?

It would be helpful to add a sentence or two at the top describing the queue naming. E.g. where do the "VA->.*" queues come from and what do they do?